src/public/backpack.base.css

```diff
@@ -841,6 +841,13 @@ body[class^='skin-purple'] .pagination>.active>span:hover {
     border-color: #f8d053;
   }
 
+  /*    FLEXBOX HELPERS */
```

Comment:
This has NOTHING to do with the current PR, but it's needed, so I say let's leave it.

Reply:
Needed by what though?